conjure-undertow-processor/src/main/java/com/palantir/conjure/java/undertow/processor/data/DefaultDecoderNames.java

```diff
@@ -216,6 +219,21 @@ private static boolean isStringMirror(TypeMirror typeMirror) {
         return ClassName.get(String.class).equals(TypeName.get(typeMirror));
     }
 
+    private static boolean isRuntimeException(TypeMirror typeMirror) {
+        if (typeMirror.getKind() != TypeKind.DECLARED) {
+            return false;
+        }
+
+        if (ClassName.get(RuntimeException.class).equals(TypeName.get(typeMirror))) {
```

Comment:
Ideally we'd use the annotation-processing `Types` object for this rather than implementing it ourselves, but if it works I'm not terribly worried :-]

Reply:
Ah sorry, didn't know about `Types`. It's a bit tricky to plumb in here since `DefaultDecoderNames` is just a utility class.

https://github.com/palantir/conjure-java/pull/1799